java11-native/.travis.yml

```diff
@@ -0,0 +1,16 @@
+#
+# Generated by: https://openapi-generator.tech
+#
+language: java
+jdk:
+  - oraclejdk11
+before_install:
+  # ensure gradlew has proper permission
+  - chmod a+x ./gradlew
+script:
+  # test using maven
+  - mvn test
+  # uncomment below to test using gradle
+  # - gradle test
+  # uncomment below to test using sbt 
+  # - sbt test
```

Comment:
Do we still use Travis for this? I can't find https://travis-ci.com/sphereon or https://travis-ci.com/sphereon-sdk